### ISSUE TITLE
Tab Corral to include tabs that are closed manually by the user

### ISSUE THREAD

**Issue by patrickbeardmore** [tabwrangler/tabwrangler#309]
I often go to the Tab Corral to look for a tab that's disappeared, and when I can't find it I remember that I must have closed it manually (via Ctrl + W or by clicking the tab X button).

I would like to be able to enable an option where all tabs that have been closed, EITHER by Tab Wrangler OR by the user, appear in the same list. This way the Tab Corral acts like an enhanced "history" for all the tabs that have closed recently.

Not everyone would want this feature all the time, so I would suggest create a choice in the "Options" tab, or having a toggle checkbox in the Tab Corral itself to include/exclude user-closed tabs.

**Comment by Zoopsoup** [comment]
Hi, is this a feature I could work on?

**Comment by ssorallen** [comment]
@Zoopsoup Definitely. Can you first write up how you think the feature should behave? A couple points I’ve considered:

* tabs closed by TW vs. tabs closed by user should look different or be labeled differently in the list
* this feature should be opt in
* the close % calculation may need to be updated
* the 1,000 tab cap might feel a lot smaller with all these tabs

**Comment by KTamas** [comment]
@Zoopsoup more information would be helpful, as outlined by @ssorallen.
